.travis.yml

```diff
@@ -28,6 +29,8 @@ before_script:
     - tar -xvf *firefox*.tar.bz2
 
 script:
+    # Run tests for each sub package
+    - firefox-ui-tests --binary firefox/firefox firefox_puppeteer/tests/manifest.ini
```

Comment:
Can we not default to a top manifest anymore? That would be preferable, I think.